internal/setup/setup.go

```diff
@@ -250,29 +254,59 @@ func ParseLayer(key string, data []byte) (*Layer, error) {
 	return &layer, err
 }
 
+var fnameExp = regexp.MustCompile("^([0-9]{3})-([a-z](?:-?[a-z0-9]){2,}).yaml$")
```

Comment:
This regex seems to prohibit a label less than 3 chars -- is that restriction necessary? For example, `123-go.yaml` or `001-a.yaml` are disallowed.

Reply:
This is intentional, in part precisely to avoid things such as `123-a.yaml`. We can always open up further if we learn this was a bad choice, but we cannot restrict further once this is released into stable, so better to err on the side of caution.